src/mqueryfront/src/query/QueryPage.js

```diff
@@ -146,8 +150,8 @@ class QueryPageInner extends Component {
     async loadMatches() {
         // Loads matches from the current page
         const OFFSET = (this.state.activePage - 1) * PAGE_SIZE;
-        const response = await axios.get(
-            `${API_URL}/matches/${this.queryHash}?offset=${OFFSET}&limit=${PAGE_SIZE}`
+        const response = await api.get(
+            `/matches/${this.queryHash}?offset=${OFFSET}&limit=${PAGE_SIZE}`
```

Comment:
😪

Reply:
unrelated to this PR, can't refactor everything at once :seal:

Reply:
Hell, might as well fix it. Just I want to stress that URL refactoring was out of scope and should've been a separate PR :P